From e20818ddea6164db45ff394c8b19d0bf3b41e734 Mon Sep 17 00:00:00 2001
From: Daniel Charles <daniel.charles@intel.com>
Date: Tue, 17 Apr 2012 15:36:59 -0700
Subject: MUST_REVERT: Don't use YV12 color format for video decoding

S/w decoder can work with RGB565 that is supported by OpenGLES implementation.
Once YV12 is supported this has to be reverted. This has to be reverted when
enabling h/w decoder as well

Change-Id: Ib7111f2a35f65c451833e05abdf08171f4a2b687
Signed-off-by: Daniel Charles <daniel.charles@intel.com>
---
 .../colorconversion/SoftwareRenderer.cpp           |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/media/libstagefright/colorconversion/SoftwareRenderer.cpp b/media/libstagefright/colorconversion/SoftwareRenderer.cpp
index 77f21b7..d30dac6 100644
--- a/media/libstagefright/colorconversion/SoftwareRenderer.cpp
+++ b/media/libstagefright/colorconversion/SoftwareRenderer.cpp
@@ -65,6 +65,7 @@ SoftwareRenderer::SoftwareRenderer(
     size_t bufWidth, bufHeight;
 
     switch (mColorFormat) {
+#if 0
         case OMX_COLOR_FormatYUV420Planar:
         case OMX_TI_COLOR_FormatYUV420PackedSemiPlanar:
         {
@@ -77,6 +78,7 @@ SoftwareRenderer::SoftwareRenderer(
 
             // fall through.
         }
+#endif
 
         default:
             halFormat = HAL_PIXEL_FORMAT_RGB_565;
-- 
1.7.4.1

